test/recipes/90-test_includes.t

```diff
@@ -0,0 +1,20 @@
+#! /usr/bin/perl
+
+use strict;
+use warnings;
+use OpenSSL::Test qw/:DEFAULT srctop_file srctop_dir with/;
+use OpenSSL::Test::Utils;
+
+setup("test_includes");
+
+plan skip_all => "test_includes doesn't work on $^O"
+    if $^O =~ /^(VMS)$/;
```

Comment:
But VMS was discussed, so is it established fact that this test doesn't work on VMS, or is it copy-paste artefact, or just "don't want to figure it out" thing? Either works for me, but I'd like to know which is it...

Reply:
I expect it to not work due to the chdir stuff. It can be fixed later I think.

Reply:
Fair enough. I suppose @levitte would have to have a look [and hopefully remove this limitation].

Reply:
May I suggest removing the restriction now?  I run daily builds based on latest nightly snapshot, and I can promise that I'll be much quicker at reacting when they fail ;-)